src/renderer/services/utils/input-utils.ts

```diff
@@ -0,0 +1,45 @@
+import {validateMnemonic} from "bip39";
+import {IInputValidity} from "../interfaces";
+
+export const KEY_WRONG_CHARACTERS_MESSAGE = "Private key must contain alphanumerical characters only";
+export const KEY_WRONG_LENGTH_MESSAGE = "Private key should have 32 bytes";
+export const INVALID_MNEMONIC_MESSAGE = "Invalid mnemonic";
+
+
+/**
+ * Method checks that input meets conditions
+ * 
+ * @param input private key or mnemonic
+ */
+export function checkValidity(input: string): IInputValidity {
+
+    const isKey = input.startsWith("0x");
+
+    const privateKey = input.split("0x")[1];
+    const hasWrongCharacters = isKey && !privateKey.match("^[a-z0-9]*$");
+
+    const wrongLength = isKey && (Buffer.from(privateKey, "hex").length !== 32);
```

Comment:
- split above into isValidPrivateKey method
- its should be in hex format so only a-f are allowed.